flaml/tune/tune.py

```diff
@@ -500,20 +514,26 @@ def easy_objective(config):
             flaml_scheduler_max_resource = max_resource
             flaml_scheduler_reduction_factor = reduction_factor
             scheduler = None
-        if lexico_objectives is None:
-            try:
-                import optuna as _
+        if lexico_objectives:
+            SearchAlgorithm = CFO
+            logger.info(
+                f"Using search algorithm {SearchAlgorithm.__name__} for lexicographic optimization. Note that when providing other search algorithms, we use CFO instead temporarily."
+            )
+            metric = lexico_objectives["metrics"][0] or DEFAULT_METRIC
+        else:
+            if not search_alg or search_alg == "BlendSearch":

```

Comment:
when `search_alg=="BlendSearch"`, we shouldn't automatically switch to CFO. We should raise an error if the import fails.

Reply:
yes. resolved.